recipes/process-main-module/src/workflow.ts

```diff
@@ -68,7 +69,14 @@ export default function transform(root: SgRoot): string | null {
 	// Step 3: Replace all code references:
 	const nodes = rootNode.findAll({
 		rule: {
-			pattern: "process.mainModule",
+			any: [
+				{
+					pattern: "process.mainModule",
+				},
+				{
+					pattern: "mainModule",
+				},
```

Comment:
We should only match this if it was found in an import / destructure

Reply:
yeah agreed and maybe have two query on to gets `mainModule` and another for the requiring

Reply:
what do you mean ? here it's only for usage part

Reply:
I created patternsToReplace to handle it